playbooks/fundamentals/index.adoc

```diff
@@ -28,6 +28,7 @@ This OpenShift and container fundamentals guide is designed to provide you with
 * link:https://www.openshift.com/promotions/for-developers.html[OpenShift for Developers^] to learn how to use OpenShift and the Wildfly application server to build and then immediately deploy a Java application.
 * link:https://www.openshift.com/promotions/docker-security.html[Docker Security: Using Containers Safely in Production^] to use Docker safely, in both production and development, you need to be aware of the potential security issues and the major tools and techniques for securing container-based systems.
 * link:https://www.openshift.com/promotions/kubernetes.html[Kubernetes: Scheduling the Future at Cloud Scale^] get an introduction to core Kubernetes concepts and components.
+* link:https://blog.openshift.com/oc-command-newbies/[Getting Started with OpenShift cli^] to learn how to use oc command.
```

Comment:
I would like to see this link listed under "OpenShift Guides" at the bottom of this doc instead.